l10n/en-US/viewer.properties

```diff
@@ -223,6 +223,7 @@ page_scale_actual=Actual Size
 page_scale_percent={{scale}}%
 
 # Loading indicator messages
+loading=Loading…
```

Comment:
You need to add this new l10n string *before* line https://github.com/mozilla/pdf.js/blob/24e598a895b712cbfa1f34deb12bb2c0f3575fee/web/l10n_utils.js#L73 as well.